### ISSUE TITLE
Implement shared [ProductItem]

### ISSUE THREAD

**Issue by hyochan** [dooboolab/whatssub#33]
[<img alt="Issuehunt badges" src="https://img.shields.io/badge/IssueHunt-%2440%20Funded-%2300A156.svg" />](https://issuehunt.io/r/dooboolab/whatssub/issues/33)

Implement shared component `ProductItem` ($40) + ($10)

![image](https://user-images.githubusercontent.com/27461460/61036036-39050c00-a403-11e9-99f6-cb4450377d50.png)

- [ ] Create `shared` component by running `dooboo shared ProductItem`.
- [ ] Use test data with below structure.
   ```
   {
     products: [
       {
         name: '',
         category: '',
         price: 0,
         currency: '',
         date: (timestamp),
         image: '',
       },
     ],
   }
   ```
- [ ] When creating `PR` please share the usage guide.
   - How to use it in `screen`?
   - Share code you wrote using your `shared` component.
- [ ] Implement `switch` toggle with [react-native-switch-toggle](https://github.com/dooboolab/react-native-switch-toggle).
   - Will be granted for extra charge if you also create `PR` for `react-native-switch-toggle` for enhancement.
- [ ] Create minimal test code
- [ ] Write a blog (extra $10) in one of [medium dooboolab](https://medium.com/@dooboolab) or [medium react-native-seoul](https://medium.com/react-native-seoul) if you prefer Korean.

** Notice **
- When the work is not done in 7days (first complete `PR`), the reward would fall down to 50%.
- When the work is not done in 14day, the work will be done by others and issue will be closed with no reward.

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Backers (Total: $40.00)

- [<img src='https://avatars0.githubusercontent.com/u/27461460?v=4' alt='hyochan' width=24 height=24> hyochan](https://issuehunt.io/u/hyochan) ($40.00)

### Tips

- Checkout the [Issuehunt explorer](https://issuehunt.io/r/dooboolab/whatssub/) to discover more funded issues.
- Need some help from other developers? [Add your repositories](https://issuehunt.io/r/new) on IssueHunt to raise funds.
---
IssueHunt has been backed by the following sponsors. [Become a sponsor](https://issuehunt.io/membership/members)
</details>

**Comment by hyochan** [comment]
@YongPilMoon  Assigned

**Comment by hyochan** [comment]
Closed via #40
